diff --git a/installlib b/installlib
index fbf2136f1..c488cc4c2 100755
--- a/installlib
+++ b/installlib
@@ -2,12 +2,13 @@
 
 set -e
 
-: "${prefix:=/usr/local}"
+: "${prefix:=/usr}"
 : "${includedir:=$prefix/include}"
 : "${libdir:=$prefix/lib}"
 : "${DESTDIR:=}"
 
 HEADER=BlocksRuntime/Block.h
+HEADER2=BlocksRuntime/Block_private.h
 LIB=libBlocksRuntime.a
 SHLIB=
 
@@ -25,6 +26,11 @@ if [ ! -r $HEADER ]; then
 	echo "Cannot find $HEADER, aborting!"
 	exit 1
 fi
+if [ ! -r $HEADER2 ]; then
+	echo "Cannot find $HEADER2, aborting!"
+	exit 1
+fi
+
 dryrun=
 if [ "$1" = '-n' ] || [ "$1" = '--dry-run' ]; then
 	dryrun=1
@@ -89,6 +95,7 @@ if [ -z "$DESTDIR" -a -z "$dryrun" -a "$myid" != 0 ]; then
 fi
 docmd "install -d "$DESTDIR"$includedir "$DESTDIR"$libdir"
 docmd "install -m 644 $HEADER "$DESTDIR"$includedir/"
+docmd "install -m 644 $HEADER2 "$DESTDIR"$includedir/"
 [ -z "$LIB" ] ||
 docmd "install -m 644 $LIB "$DESTDIR"$libdir/"
 [ -z "$SHLIB" ] ||
